test/uk/gov/hmrc/plasticpackagingtax/registration/forms/group/RemoveMemberSpec.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.forms.group
+
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import uk.gov.hmrc.plasticpackagingtax.registration.forms.CommonFormValues
+
+class RemoveMemberSpec extends AnyWordSpec with Matchers with CommonFormValues {
+
+  "Remove Member Form" should {
+    "round trip" when {
```

Comment:
do we want to test other bindings for this form?

Reply:
I think this is sufficient - there's a balance to be struck between testing absolutely everything and testing enough to have sufficient confidence that the code operates as expected.